entrypoint.sh

```diff
@@ -31,21 +32,61 @@ secret_not_found() {
   exit 1
 }
 
+WRANGLER_VERSION=2
+
+# If no Wrangler version is specified install v2.
+if [ -z "$INPUT_WRANGLERVERSION" ]; then
+  npm i -g wrangler
+
+# If Wrangler version starts with 2 then install wrangler v2
+elif [[ "$INPUT_WRANGLERVERSION" == 1* ]]; then
+  npm i -g "@cloudflare/wrangler@$INPUT_WRANGLERVERSION"
+  WRANGLER_VERSION=1
+
+# Else install Wrangler v2
+else
+  npm i -g "wrangler@$INPUT_WRANGLERVERSION"
+  WRANGLER_VERSION=1
+fi
+
 # If an API token is detected as input
-if [ -n "$INPUT_APITOKEN" ]
-then
-  export CF_API_TOKEN="$INPUT_APITOKEN"
+if [ -n "$INPUT_APITOKEN" ]; then
+
+  # Wrangler v1 uses CF_API_TOKEN but v2 uses CLOUDFLARE_API_TOKEN
```

Comment:
Wrangler 2 still supports the `CF_...` environment vars for the time being. 
So you could avoid these checks if you like.

Reply:
Yeah, I just wanted to avoid any possible future breakage and the deprecation warnings. Not a big deal doing it this way